activities/member-relations/index.md

```diff
@@ -4,4 +4,74 @@ type: Index
 
 # Member relations
 
+The Foundation for Public Code is a member-owned association. Only wholly publicly owned organizations may become members. Members participate in the steering and governance of the association by voting at the general assembly. They also participate by financing the association through yearly dues.
+
 We support members in getting the most from their membership and [codebase stewardship](../codebase-stewardship), as well as building and maintaining relationships with and between members.
+
+The process towards full membership is roughly divided into three phases, noting that every organization we speak to is unique. This means that individual membership  processes may differ between the organizations we work with.
+
+## 1. Membership growth
+
+During member growth we connect with and support wholly publically-owned organizations in their path towards membership.
+
+In this phase, we aim to:
+
+* build trust in our organization and our processes
+* determine focal and contact points within the prospective member organization
+* gain insight into the member's organizational structures and how codebase stewardship can benefit them
+
+The following are examples of activities we may conduct during member acquisition:
+
+* Conduct workshops to raise awareness about
+  * Codebase stewardship
+  * Public code
+  * The Standard for Public Code
+  * Our organizational model
+* Hold meetings to exchange knowledge and support with contracting
+* Have regular calls to answer any questions and ensure continuous, open communication
+
+Member growth is finalized when the public organization signs a [membership contract](../membership-growth/membership-contract.md).
+
+## 2. Member onboarding
+
+During member onboarding we help new members get accustomed to our processes, and support their public code projects.
+
+In this phase, we aim to:
+
+* support members with their public code projects and codebases
+* introduce members to our network and codebase communities
+* prepare members for their first general assembly
+
+The following are examples of activities we may conduct during member onboarding:
+
+* Develop a membership plan to identify members' support needs, and a timeline of milestones
+* Assess codebases for codebase stewardship
+* Conduct trainings on various topics surrounding public code
+* Conduct a [membership workshop](../membership-growth/founding-membership-workshop.md) to:
```

Comment:
This workshop needs updating to no longer say 'founding member'.

Reply:
Will do!